src/Smart.Design.Library/TagHelpers/Box/IBoxHtmlGenerator.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Smart.Design.Library.TagHelpers.Box;
+
+/// <summary>
+/// Generate a box with a title, sub title and list of labels and values.
+/// </summary>
+public interface IBoxHtmlGenerator
+{
+    /// <summary>
+    /// Generate a list of value from a <see cref="labelAndLinks"/> dictionnary
```

Comment:
Si c'est une liste de labels et values, alors peut-être appeler ça labelsAndValues au lieu de valuesAndLinks?

Sinon:

```suggestion
    /// Generate a list of values from a <see cref="labelsAndLinks"/> dictionnary
```